src/Microsoft.ML.CodeGenerator/CodeGenerator/CSharp/CodeGenerator.cs

```diff
@@ -89,6 +90,53 @@ public void GenerateOutput()
             Utils.AddProjectsToSolution(modelprojectDir, modelProjectName, consoleAppProjectDir, consoleAppProjectName, solutionPath);
         }
 
+        public void GenerateAzureRemoteImageOutput()
```

Comment:
Ideal way is we don't differentiate pipelines from AutoML / Azure AutoML, However there's already too many if-else conditions and some function's parameter list are so long to be bearable. So I feel like make it a special case can be a better choice